config/locales/en.yml

```diff
@@ -652,18 +652,18 @@ en:
       title: File not found
       description: Couldn't find a file/directory/API operation by that name on the OpenStreetMap server (HTTP 404)
   friendships:
-    make_friend:
-      heading: "Add %{user} as a friend?"
-      button: "Add as friend"
-      success: "%{name} is now your friend!"
-      failed: "Sorry, failed to add %{name} as a friend."
-      already_a_friend: "You are already friends with %{name}."
-      limit_exceeded: "You have friended a lot of users recently. Please wait a while before trying to friend any more."
-    remove_friend:
-      heading: "Unfriend %{user}?"
-      button: "Unfriend"
-      success: "%{name} was removed from your friends."
-      not_a_friend: "%{name} is not one of your friends."
+    follow_user:
+      heading: "Do you want to follow %{user}?"
+      button: "Follow User"
+      success: "You are now following %{name}!"
+      failed: "Sorry, failed to follow %{name}."
```

Comment:
Suggesting one more alternative: "Sorry, your request for following %{name} has failed." Wording sounds a bit unnatural here imho

Reply:
Wording was changed according to the suggestion.